pkg/auth/migrate/rbac_to_acl.go

```diff
@@ -114,6 +118,35 @@ func RBACToACL(ctx context.Context, svc auth.Service, doUpdate bool, creationTim
 		messageFunc(group.DisplayName, *newACL, warnings)
 	}
 
+	users, _, err := svc.ListUsers(ctx, &model.PaginationParams{Amount: maxUsers + 1})
+	if err != nil {
+		return fmt.Errorf("list users: %w", err)
+	}
+	if len(users) > maxUsers {
+		return fmt.Errorf("%w (got %d)", ErrTooManyUsers, len(users))
+	}
+	for _, user := range users {
+		var warnings error
+		policies, _, err := svc.ListUserPolicies(ctx, user.Username, &model.PaginationParams{Amount: maxUserPolicies + 1})
+		if err != nil {
+			return fmt.Errorf("list users policies: %w", err)
+		}
+		if len(policies) > maxUsers {
+			return fmt.Errorf("%w (got %d)", ErrTooManyUsers, len(policies))
+		}
```

Comment:
it seems that you're counting the number of policies against the maximum number of users, I believe that you meant to use the `maxUserPolicies` ?
```suggestion
		if len(policies) > maxUserPolicies {
			return fmt.Errorf("%w (got %d)", ErrTooManyPolicies, len(policies))
		}
```